cmd/ipfs/init.go

```diff
@@ -36,6 +36,8 @@ Available profiles:
         running IPFS on machines with public IPv4 addresses.
     'test' - Reduces external interference of IPFS daemon, this
         is useful when using the daemon in test environments.
+    'lowpower' - Reduces daemon overhead on the system. May affect node
```

Comment:
Maybe `'laptop'`? Honestly, laptop users should use this (less problems with random WiFis, fewer ephemeral DHT nodes, etc. Alternatively, we could have a separate laptop mode that occasionally reprovides pin roots.

Reply:
Probably the second option, I meant this setting for mobile devices (Android phones, etc.)

Reply:
"May affect node functionality" should be better described to tell what functionality it would affect. Seemingly, it seems to run the dht in client mode and also set really low ConnMgr values, meaning it would probably take longer to find and download content. Would be nice to add that to the description.

Reply:
done